### ISSUE TITLE
[Feature Request] storage-stats on 'snapshot create'

### ISSUE THREAD

**Issue by tribor** [kopia/kopia#2667]
What are you trying to do? What problem would this solve?
---------
I would like to write metrics from the latest `snapshot create` command including the amount of **uploaded bytes and files**.  My goal is to export these metrics to prometheus (via textfile exporter). I'm aware the kopia has already an exporter but I rather would like to generate my own metrics for that.

What should kopia do differently? Which functionality do you think we should add?
---------
The output of `kopia snapshot create --json <data>` contains valuable information about the _processed_ data. But it is missing the information how much bytes and files have been _added_ to the repo. This information however is displayed in the non-json ouput:
```
 * 0 hashing, 14 hashed (545.5 KB), 2111 cached (149.5 MB), uploaded 0 B, estimating...
```
Maybe kopia could add `--storage-stats` to the snapshot command or always include this information by default in the json output? 

Output of `kopia --version`
---------
0.12.1 build: 5227d74996b6520f9f96e4203cfe00b832a60d5f from: kopia/kopia

other information
---------
related to #609 [Feature Request] Prometheus metrics from kopia server enhancement

**Comment by tribor** [comment]
any news?

**Comment by xxxliqu1dxxx** [comment]
There's already a flag on snapshot list called --storage-stats.
You can use it like this ./kopia snapshot list --storage stats.
It will show things like this:
`2022-07-14 16:42:31 EDT kcd9b44fa5c2854b6d2240cb4fd631942 301 GB drwxrwxrwx files:11335 dirs:1553 new-data:19.9 GB new-files:1805 new-dirs:318 compression:28.1% (latest-1,hourly-1,daily-1,weekly-1,monthly-1,annual-1)`

**Comment by tribor** [comment]
Yes, but this only would be a _workaround_ for my feature request. I want to have a direct output just after the backup was run to scrape this information for my backup metrics.

**Comment by tribor** [comment]
the ultimate goal is to create a textfile (for prometheus scraping) from the kopia backup run output like this:
```
kopia_backup_seconds{jobname="server01"} 2
kopia_backup_files_new{jobname="server01"} 0
kopia_backup_data_added{jobname="server01"} 707
kopia_backup_total_files_processed{jobname="server01"} 7
kopia_backup_total_bytes_processed{jobname="server01"} 4904
kopia_backup_reposize{jobname="server01"} 19659
```
